<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sigmaf</class>
 <widget class="QDockWidget" name="sigmaf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saisie d'un sigma facies</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-193</y>
         <width>420</width>
         <height>838</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Série</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_nom_sf">
             <property name="text">
              <string>Code de la série :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="code_serie">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_nom_sf_2">
             <property name="text">
              <string>Libellé de la série :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="lb_serie">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Type de complexe : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="type_cplx">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Type de série : </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="type_serie">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Confiance série :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="cfc_serie">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Sigma facies</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_nom_sf_4">
             <property name="text">
              <string>Code du sigma facies :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="code_sigma">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_nom_sf_3">
             <property name="text">
              <string>Libellé du sigma facies :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="lb_sigma">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Type de facies : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="typ_facies">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="rmq_typ_fa"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Typicité facies</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="typicite">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QTextEdit" name="rmq_typcte"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Saturation phytocenotique : </string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="sat_phy">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QTextEdit" name="rmq_sat_ph"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Confiance facies :</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="cfc_facies">
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Pourcentage de recouvrement</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="1" column="0">
            <widget class="QLineEdit" name="pct_recouv">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="rel_composyntaxon">
          <property name="title">
           <string>Composition réelle en syntaxons</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tbl_syntax">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="qwidget" native="true">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QPushButton" name="add">
                <property name="text">
                 <string>Ajouter un syntaxon</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="edit">
                <property name="text">
                 <string>Modifier</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="delt">
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="valid_btn">
          <property name="text">
           <string>Valider</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>code_serie</tabstop>
  <tabstop>lb_serie</tabstop>
  <tabstop>type_cplx</tabstop>
  <tabstop>type_serie</tabstop>
  <tabstop>cfc_serie</tabstop>
  <tabstop>code_sigma</tabstop>
  <tabstop>lb_sigma</tabstop>
  <tabstop>typ_facies</tabstop>
  <tabstop>rmq_typ_fa</tabstop>
  <tabstop>typicite</tabstop>
  <tabstop>rmq_typcte</tabstop>
  <tabstop>sat_phy</tabstop>
  <tabstop>rmq_sat_ph</tabstop>
  <tabstop>cfc_facies</tabstop>
  <tabstop>pct_recouv</tabstop>
  <tabstop>tbl_syntax</tabstop>
  <tabstop>add</tabstop>
  <tabstop>edit</tabstop>
  <tabstop>delt</tabstop>
  <tabstop>valid_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
